apps/playground-web/src/app/connect/pay/page.tsx

```diff
@@ -101,44 +105,98 @@ function StyledPayEmbed() {
   );
 }
 
-function PayTransactionButton() {
+function BuyMerch() {
+  return (
+    <>
+      <div className="space-y-2">
+        <h2 className="text-2xl sm:text-3xl font-semibold tracking-tight">
+          Direct Payments
+        </h2>
+        <p className="max-w-[600px]">
+          Take paymets from Fiat or Crypto directly to your seller wallet.
+          <br />
+          Get notified for every sale through webhooks, which lets you trigger
+          any action you want like shipping physical goods, activating services
+          or doing onchain actions.
+        </p>
+      </div>
+
+      <CodeExample
+        preview={<BuyMerchPreview />}
+        code={`import { PayEmbed } from "thirdweb/react";
+
+        function App() {
+          return (
+            <PayEmbed
+              client={client}
+              theme={"light"}
+              payOptions={{
+                mode: "direct_payment",
+                paymentInfo: {
+                  amount: "35",
+                  chain: base,
+                  currency: USDC,
+                  sellerAddress: "0xEb0effdFB4dC5b3d5d3aC6ce29F3ED213E95d675",
```

Comment:
We should probably also show how we get `base` and `USDC` here

Reply:
how do you feel about exporting some default tokens?